packages/hardhat-zksync-deploy/src/task-actions.ts

```diff
@@ -0,0 +1,6 @@
+import { HardhatRuntimeEnvironment, TaskArguments } from 'hardhat/types';
+import { callDeployScripts } from './plugin';
+
+export async function zkSyncDeploy(taskArgs: TaskArguments, hre: HardhatRuntimeEnvironment) {
+    await callDeployScripts(hre, taskArgs.script);
+}
```

Comment:
I'm not sure why we should extract a one-line function invocation into a separate file. To me, this just adds unnecessary indirection.

But if you prefer it this way, I don't mind

Reply:
Same reason here, the codebase will grow and very soon this will not be just one-line function or we will have more task actions, so in my opinion this is a good practice, and I would prefer to keep it like this if you are ok with that.